### PR TITLE
Returning angle using tan(dw/dh) for scanning and hough

### PR DESCRIPTION
**Pull request by abhigyandabla**

### CODE REVIEW

File: algorithms/HoughAlgorithm.py [hunk 1]
```diff
@@ -73,16 +75,20 @@ def processFrame(self, frame, show=True):
         intersections, points = Lines.getIntersections(lines)
         vPoint = Lines.drawVanishingPoint(lineimg, points)
 
+        dwVP = vPoint[0] - (self.WIDTH // 2)
+        dhVP = vPoint[1]
```
Comment (L0Lmaker) [inline algorithms/HoughAlgorithm.py]: make variables more descriptive

File: algorithms/HoughAlgorithm.py [hunk 2]
```diff
@@ -73,16 +75,20 @@ def processFrame(self, frame, show=True):
...
-
-        return lineimg, vPoint
+        # if show:
+        #     cv2.imshow('frame', frame)
+        #     cv2.imshow('mask_hough',mask)
+        #     cv2.imshow('edges', edges)
+        #     cv2.imshow('hough algorithm', lineimg)
+        #     cv2.waitKey(1)
+
+        return lineimg, angle
```
Comment (L0Lmaker) [inline algorithms/HoughAlgorithm.py]: Check if this is used by a command line argument, if not lets add it. Could be used by `test_algorithms.py`.

File: algorithms/MiniContoursAlgorithm.py [hunk 3]
```diff
@@ -187,10 +187,10 @@ def getCenterHoughLines(self, frame, num_strips=60, lines_max=30, threshold=4, m
...
 
-        cv2.imshow('frame', frame)
-        cv2.imshow('mask', mask)
-        cv2.imshow('c_mask', c_mask)
-        cv2.imshow('points', points)
+        # cv2.imshow('frame', frame)
+        # cv2.imshow('mask', mask)
+        # cv2.imshow('c_mask', c_mask)
+        # cv2.imshow('points', points)
         # cv2.waitKey(1)
```
Comment (L0Lmaker) [inline algorithms/MiniContoursAlgorithm.py]: Perhaps we can add the show logic here?

File: algorithms/ScanningAlgorithm.py [hunk 4]
```diff
@@ -73,7 +74,7 @@ def create_line(self, start_x, start_y, end_x, end_y):
         return line
 
     def processFrame(self, frame, show=False):
-        cv2.imshow('original frame', frame)
+        #cv2.imshow('original frame', frame)
```
Comment (L0Lmaker) [inline algorithms/ScanningAlgorithm.py]: Same as before.

File: algorithms/ScanningAlgorithm.py [hunk 5]
```diff
@@ -141,18 +142,19 @@ def processFrame(self, frame, show=False):
...
-            cv2.imshow('mask', mask)
+        # Commenting out because angle is not consistent with other algos
+        # # finding the angle between the center of the frame and the line drawn to the vanishing point
+        # up = [0, 1]
+        # dir = [self.WIDTH // 2 - vanishing_point[0], self.mid_y - self.upper_y_bound]
+        # angle = np.arccos(np.dot(up, dir) / (np.linalg.norm(up) * np.linalg.norm(dir))) * 180 / np.pi
+
+        dwVP = vanishing_point[0] - (self.WIDTH // 2)
+        dhVP = vanishing_point[1]
+        angle = round(math.degrees(math.atan(dwVP/dhVP)), 2)
```
Comment (L0Lmaker) [inline algorithms/ScanningAlgorithm.py]: same as before

**Review by L0Lmaker — CHANGES_REQUESTED** [review]

File: algorithms/HoughAlgorithm.py (hunk 2 above)
Comment (abhigyandabla) [reply algorithms/HoughAlgorithm.py]: It is not a cli argument rn, we can add it later

**Pull request closed without merging**